liberty-managed/src/main/java/io/openliberty/arquillian/managed/WLPManagedContainer.java

```diff
@@ -1059,7 +1064,12 @@ private String createDeploymentName(String archiveName)
 
    private String createDeploymentType(String archiveName)
    {
-      return archiveName.substring(archiveName.lastIndexOf(".")+1);
+      String depType = archiveName.substring(archiveName.lastIndexOf(".")+1);
+      if (depType.equals("jar")) {
```

Comment:
This part concerns me since we support deploying a SpringBoot `jar` in the Liberty Maven plugin...that is specified with  the `spring-boot-project` value on the `deploy` goal for `<deployPackages>`. Right now are you saying that Liberty Arquillian does not support SpringBoot `jar` so it is okay to presume a deployment type of `ejb` if the archive is a `jar`?